Automattic/pocket-casts-ios
# UpNext on tab bar

Fixes #

This PR implements the display of the UpNext on the tab bar and showing the profile link on the podcasts tab.

https://github.com/Automattic/pocket-casts-ios/assets/651601/ce5c13c3-1b0f-4716-90b3-639a1b3aaeef

| Light |  Dark |  Rose |
| - | - | - |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-04-14 at 14 04 38](https://github.com/Automattic/pocket-casts-ios/assets/651601/b2f9ea21-412f-4498-a02a-3bb0b78e805d) |  ![Simulator Screenshot - iPhone 15 Pro - 2024-04-14 at 14 05 00](https://github.com/Automattic/pocket-casts-ios/assets/651601/97be0415-fc5d-422e-bf03-3c870e57cbfb) | ![Simulator Screenshot - iPhone 15 Pro - 2024-04-14 at 14 05 17](https://github.com/Automattic/pocket-casts-ios/assets/651601/c28ee930-f8ec-4f20-8c17-719ee2cf3385) |
|  ![Simulator Screenshot - iPhone 15 Pro - 2024-04-14 at 16 04 35](https://github.com/Automattic/pocket-casts-ios/assets/651601/da555809-8569-437c-86d6-8d26f2250c98) | ![Simulator Screenshot - iPhone 15 Pro - 2024-04-14 at 16 04 18](https://github.com/Automattic/pocket-casts-ios/assets/651601/83d06003-d26e-456b-b821-2653ce17bd0e) | ![Simulator Screenshot - iPhone 15 Pro - 2024-04-14 at 16 04 03](https://github.com/Automattic/pocket-casts-ios/assets/651601/2379070e-1f68-42ac-b493-23f535e92407) |


## To test

 - Start the app
 - Ensure that the FF UpNext on the tab bar is enabled
 - Restart the app if you need to enable it
 - Check if Up Next shows on the tab on the second position
 - Tap on it
 - Check if the Up Next shows correctly and you can interact with the item correctly
 - Go to the Podcasts tab
 - Check if on the top left you see the profile icon with the user avatar ( the default avatar will be show if the user is not logged in or does not have an avatar set)
 - Tap on the profile button
 - Check that the profile view shows up by being pushed to the navigation of podcasts
 - Test the navigation to profile with this navigation
   - Open the bookmarks screens when you don't have any bookmarks, tap on headphone settings and check if it navigates correctly  

-Check analytics events
 - Enable tracks logging on Profile -> Beta Developer
 - Tap on UpNext on the tab bar
 - Check if you see: `up_next_tab_opened`

## Checklist


When updating the spreadsheet I noticed that the format for tab events in the the `[tab name]_tab_opened` but on the app we are using this format `[tab name]_tab_open`. What is the correct one?
